jekyll/_cci2/container-runner-installation.adoc

```diff
@@ -12,40 +10,63 @@ contentTags:
 :icons: font
 :toc: macro
 :toc-title:
+:container:
 
 This page describes how to install CircleCI's container runner.
 
-{% include snippets/runner-platform-prerequisites.adoc %}
+// {% include snippets/runner-platform-prerequisites.adoc %}
 
-[#installation-instructions]
-== Installation instructions
+[#prerequisites]
+== Prerequisites
 
-1. Add the container runner Helm repo by running `helm repo add container-agent https://packagecloud.io/circleci/container-agent/helm`
+{% include snippets/runner/container-runner-prereq.adoc %}
 
-2. Run `helm repo update`
+[#self-hosted-runner-terms-agreement]
+== Self-hosted runner terms agreement
 
-3. Run `kubectl create namespace circleci` to create the `circleci` Kubernetes namespace
+{% include snippets/runner/terms.adoc %}
 
-4. Create a file called `values.yaml` file containing the following:
+[#create-namespace-and-resource-class]
+== 1. Create namespace and resource class 
+
+[.tab.container-runner.Web_app_installation]
+--
+{% include snippets/runner/install-with-web-app-steps.adoc %}
+--
+[.tab.container-runner.CLI_installation]
+--
+{% include snippets/runner/install-with-cli-steps.adoc %}
+--
+
+[#container-runner-installation]
+== 2. Container runner installation
+
+. Add the container runner Helm repo by running `helm repo add container-agent https://packagecloud.io/circleci/container-agent/helm`
+
+. Run `helm repo update`
+
+. Run `kubectl create namespace circleci` to create the `circleci` Kubernetes namespace
```

Comment:
I think it's better for the reader to have these commands we want them to run in code blocks rather than inline, so they are easy to copy and paste